### PR TITLE
Add Picture-in-Picture sample

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Note that the Origin Trial link is working but PiP is not available yet in the form.

Try it at https://beaufortfrancois.github.io/samples/picture-in-picture/

R: @mounirlamouri @japacible

### CODE REVIEW

File: picture-in-picture/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<p>
+The <a href="https://wicg.github.io/picture-in-picture/">Picture-in-Picture
+API</a> allows websites to create a floating video window that is always on top
+of other windows so that users may continue consuming media while they interact
+with other sites or applications on their device.
+</p>
+<p>
+Note: In Chrome 68, it is <a
+href="http://bit.ly/PipOriginTrials">available for trials</a> so
+that websites can use it without any experimental flag.
```
Comment (japacible) [inline picture-in-picture/index.html]: s/flag/flags

File: picture-in-picture/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+</p>
+
+<p>
+Credits: Media files are © copyright Blender Foundation | <a href="http://www.blender.org">www.blender.org </a>.
```
Comment (japacible) [inline picture-in-picture/index.html]: Remove extra space before </a>.

**Review by japacible — APPROVED** [review]
Cool to play with this! :')

lgtm

File: picture-in-picture/index.html (hunk 2 above)
Comment (beaufortfrancois) [reply picture-in-picture/index.html]: Done.

File: picture-in-picture/index.html (hunk 1 above)
Comment (beaufortfrancois) [reply picture-in-picture/index.html]: Done.

**Comment by beaufortfrancois** [comment]
Thanks @japacible. I've addressed your feedback.

**Review by mounirlamouri — APPROVED** [review]
lgtm! :)

**Comment by beaufortfrancois** [comment]
I'll merge now so that developers can try it.

**Pull request merged**